lib/kitchen/transport/winrm.rb

```diff
@@ -389,24 +410,38 @@ def to_s
       # @api private
       def connection_options(data)
         opts = {
-          :instance_name          => instance.name,
-          :kitchen_root           => data[:kitchen_root],
-          :logger                 => logger,
-          :winrm_transport        => :plaintext,
-          :disable_sspi           => true,
-          :basic_auth_only        => true,
-          :endpoint               => data[:endpoint_template] % data,
-          :user                   => data[:username],
-          :pass                   => data[:password],
-          :rdp_port               => data[:rdp_port],
-          :connection_retries     => data[:connection_retries],
+          :instance_name => instance.name,
+          :kitchen_root => data[:kitchen_root],
+          :logger => logger,
+          :endpoint => data[:endpoint_template] % data,
+          :user => data[:username],
+          :pass => data[:password],
+          :rdp_port => data[:rdp_port],
+          :connection_retries => data[:connection_retries],
           :connection_retry_sleep => data[:connection_retry_sleep],
-          :max_wait_until_ready   => data[:max_wait_until_ready]
+          :max_wait_until_ready => data[:max_wait_until_ready],
+          :winrm_transport => data[:winrm_transport]
         }
-
+        opts.merge!(additional_transport_args(opts[:winrm_transport]))
         opts
       end
 
+      def additional_transport_args(transport_type)
+        case transport_type
+        when :ssl, :sspinegotiate
```

Comment:
I suggest we match the logic in knife-windows 1.x:
`:disable_sspi` if not on windows, using `ssl, or not not authenticating with `negotiate`
`:basic_auth_only` only from linux when ssl is not specified